pkg/virt-controller/watch/migration.go

```diff
@@ -731,8 +731,21 @@ func (c *MigrationController) handleTargetPodHandoff(migration *virtv1.VirtualMa
 func (c *MigrationController) handleSignalMigrationAbort(migration *virtv1.VirtualMachineInstanceMigration, vmi *virtv1.VirtualMachineInstance) error {
 
 	vmiCopy := vmi.DeepCopy()
+	now := v1.NewTime(time.Now())
```

Comment:
Yes, it is a must since previously we assumed that if we reach this function it means the migration is running. This has several implications, for example `vmiCopy.Status.MigrationState` might be nil.